.github/workflows/update_formula.yml

```diff
@@ -0,0 +1,46 @@
+name: Homebrew update formula
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        required: true
+        description: "fp release version (just numbers without the v, e.g.: 2.2.0)"
+        type: string
+      sha_aarch64_apple:
```

Comment:
I recommend to keep this variables the same as the environment variables used in the template (expect upper/lower casing).